docs/src/tutorial/example-data.rst

```diff
@@ -0,0 +1,87 @@
+Run the workflow using example data
+===================================
+
+In this tutorial, you will run the workflow using example reference data.
```

Comment:
I think we could expand this a bit to give a better understanding for _why_ we are asking users to do this. Something like:

> The aim of this first tutorial is to introduce our SARS-CoV-2 workflow.
To do this, we will run the workflow using a small set of reference data which we provide.
This tutorial leads on to subsequent tutorials where we will walk through more complex scenarios.

Reply:
Done: 35331f4f